soh/src/overlays/actors/ovl_En_Box/z_en_box.c

```diff
@@ -767,7 +768,16 @@ void EnBox_UpdateSizeAndTexture(EnBox* this, PlayState* play) {
 }
 
 void EnBox_CreateExtraChestTextures() {
-    if (hasCreatedRandoChestTextures) return;
+    if (hasCreatedRandoChestTextures || hasCustomChestDLs) return;
+
+    // Don't patch textures for custom chest models, as they do not import textures the exact same way as vanilla chests
```

Comment:
Does this handle the case of the custom models being toggle-able by alt? I don't think so if I'm understanding this correctly, or maybe it's a non-issue for another reason

Reply:
ah good catch, probably not since its a one time look up.

Reply:
ok rearranged some things so it checks for the model being custom on every update instead of a one-time check.
Not a fan of having to check every update, but its fine for now.